Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityPointer.cs

```diff
@@ -116,6 +116,16 @@ public interface IMixedRealityPointer : IEqualityComparer
         /// </summary>
         void OnPreSceneQuery();
 
+        /// <summary>
+        /// Called after performing the scene query. Updates hitinfo
+        /// </summary>
+        bool SceneQuery(LayerMask[] prioritizedLayerMasks, bool focusIndividualCompoundCollider, out MixedRealityRaycastHit hitInfo);
```

Comment:
Also hadn't realized this would be a breaking interface change...since this is a long-running interface, I'd be _really_ hesitant to break it, as there could be any number of custom pointers created out there. Is there any way to treat our sphere pointer specially while not requiring every pointer to implement these two methods? Is there any benefit to this change in pattern to anything except the sphere pointer?

Reply:
The change in pattern is only for the sphere pointer, and yeah I admit this was one of the changes I was most hesitant on. There doesn't seem to be a way to check if the pointer is specifically a spherepointer (due to how our asmdefs are structured)